app/src/main/java/org/oppia/app/topic/train/SkillSelectionAdapter.kt

```diff
@@ -0,0 +1,53 @@
+package org.oppia.app.topic.train;
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.databinding.library.baseAdapters.BR
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.topic_train_skill_view.view.*
+import org.oppia.app.R
+import org.oppia.app.databinding.TopicTrainSkillViewBinding
+
+// TODO(#172): Make use of generic data-binding-enabled RecyclerView adapter.
+/** Adapter to bind skills to [RecyclerView] inside [TopicTrainFragment]. */
+class SkillSelectionAdapter(
+  private val skillList: List<String>,
+  private val skillInterface: SkillInterface
+) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+
+  override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
+    val inflater = LayoutInflater.from(parent.context)
+    val binding =
+      DataBindingUtil.inflate<TopicTrainSkillViewBinding>(
+        inflater,
+        R.layout.topic_train_skill_view,
+        parent,
+        false
```

Comment:
Nit: please add in-line comment with the argument name corresponding to this literal to provide context to readers, e.g.:

/* attachToParent= */ false

Reply:
Done.